### PR TITLE
'model, ddl were changed, dml was created'

### PR DESCRIPTION
**Pull request by opapina**

### CODE REVIEW

File: src/main/resources/dml.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 use school_timetable;
 #inserts into DB
 
-insert into day_names(name) values('Monday'), ('Tuesday'), ('Wednesday'), ('Thursday'), ('Friday'), ('Saturday'), ('Sunday');                                     
+insert into days(name) values('Monday'), ('Tuesday'), ('Wednesday'), ('Thursday'), ('Friday'), ('Saturday'), ('Sunday');
```
Comment (amuravski) [inline src/main/resources/dml.sql:4]: insert into days(name) values('Monday'),
('Tuesday'),
('Wednesday'),
...
format all other insert like this

File: src/main/resources/dml.sql [hunk 2]
```diff
@@ -15,19 +15,19 @@ insert into teachers(subject_id, full_name) values (1, 'Maria Ivanova'), (1, 'El
...
                                        (15, 'Maria Slavkina'), (16, 'Elena Leniva'), (16,'Каterina Ivanova'), (17, 'Ivan Petrov');    
 insert into school_timetables(hashcode) values (1), (2);
-insert into class_names (name) values ('7A'), ('8B'), ('1A'), ('2B'), ('3A'), ('5B'), ('6A'), ('10B');
+insert into school_classes (name) values ('7A'), ('8B'), ('1A'), ('2B'), ('3A'), ('5B'), ('6A'), ('10B');
 insert into class_timetables(class_id, school_timetable_id) values (1, 2), (1,1), (2, 2), (2,1);
-insert into days (timetable_id, day_name_id) values (1, 2), (2, 2), (3, 2), (4, 2);
-insert into lessons (teacher_id,day_id,lesson_number) values (2, 1, 1), (3, 2, 2), (4, 3, 3), (5, 4, 4), (6, 5, 5),
-(17, 1, 1), (7, 2, 2), (8, 3, 3), (9, 4, 4), (10, 5, 5), (11, 1, 1), (13, 2, 2), (14, 3, 3), (15, 4, 4), (16, 5, 5); 
+insert into school_days (timetable_id, day_id) values (1, 2), (2, 2), (3, 2), (4, 2);
+insert into lessons (teacher_id,school_day_id,lesson_number) values (2, 1, 1), (3, 2, 2), (4, 3, 3), (5, 4, 4), (6, 4, 5),
```
Comment (amuravski) [inline src/main/resources/dml.sql:21]: fix spaces around commas

**Review by amuravski — CHANGES_REQUESTED** [review]
fix everything described in the review

File: src/main/resources/dml.sql [hunk 3]
```diff
@@ -1,7 +1,7 @@
 use school_timetable;
 #inserts into DB
 
-insert into day_names(name) values('Monday'), ('Tuesday'), ('Wednesday'), ('Thursday'), ('Friday'), ('Saturday'), ('Sunday');                                     
+insert into days(name) values('Monday'), ('Tuesday'), ('Wednesday'), ('Thursday'), ('Friday'), ('Saturday'), ('Sunday');                                     
 			
 insert into subjects(name) values ('chemistry'), ('computing'), ('drawing'), ('economy'), ('geometry'), ('language_by'),
 								  ('language_de'), ('language_en'), ('literature_by'), ('literature_de'), ('literature_en'),
```
[the remark this answers is not in the document]
Comment (opapina) [reply src/main/resources/dml.sql]: was fixed

File: src/main/resources/dml.sql (hunk 2 above)
Comment (opapina) [reply src/main/resources/dml.sql]: was fixed

**Review by amuravski — APPROVED** [review]

**Pull request merged**